contact/location.go

```diff
@@ -0,0 +1,53 @@
+package contact
+
+import (
+	"encoding/json"
+
+	model "github.com/owasp-amass/open-asset-model"
+)
+
+// This struct represents the location linked to the possible asset
+type Location struct {
+	// FormattedAddress is the human-readable address of the location.
+	FormattedAddress string `json:"formatted_address,omitempty"`
+
+	// BuildingNumber is the number of the building at the location.
+	BuildingNumber string `json:"building_number,omitempty"`
+
+	// StreetName is the name of the street at the location.
+	StreetName string `json:"street_name,omitempty"`
+
+	// Unit is the unit number of the location.
+	Unit string `json:"unit,omitempty"`
+
+	// Building is the name or number of the building at the location.
+	Building string `json:"building,omitempty"`
+
+	// Town is the name of the town or city at the location.
+	Town string `json:"town,omitempty"`
+
+	// Locality is the name of the locality at the location.
+	Locality string `json:"locality,omitempty"`
+
+	// Region is the name of the region or state at the location.
+	Region string `json:"region,omitempty"`
+
+	// CountryCode is the ISO 3166-1 alpha-2 country code of the location.
+	CountryCode string `json:"country_code,omitempty"`
+
+	// PostalCode is the postal code of the location.
+	PostalCode string `json:"postal_code,omitempty"`
+
+	// LatLong is the latitude and longitude of the location.
+	LatLong [2]float64 `json:"lat_long,omitempty"`
```

Comment:
@The-Inceptions regarding geo data, I think we should leave this out until we research this further and can model it appropriately.

For example, you can represent geo's as a lat+long point, lat+long+radius (fencing), or geometric shape (e.g. a government defined municipality)